backend/tools/event_sender/src/main.rs

```diff
@@ -115,40 +143,61 @@ async fn app() -> anyhow::Result<()> {
     };
 
     let target = cli.count * cli.burst;
-    let sleep_duration = Duration::from_millis(cli.latency as u64);
+
+    // Random number generator used to product random intervals
+    fn random_num(from: usize, to: usize) -> usize {
```

Comment:
This should be inlined to just `rand::thread_rng().gen_range(from..to)` as there is only one use of it

Reply:
We were using it for something else too initially but will update.